core/whistle_media-1.0.0/src/wh_media_map.erl

```diff
@@ -362,7 +362,10 @@ get_map(?WH_MEDIA_DB = Db, PromptId) ->
             #media_map{id=MapId
                        ,account_id=Db
                        ,prompt_id=PromptId
-                       ,languages=wh_json:new()
+                       ,languages=wh_json:set_value(wh_media_util:default_prompt_language()
+                                                    ,<<"/", ?WH_MEDIA_DB/binary, "/", (wh_media_util:default_prompt_language())/binary, "%2F", PromptId/binary>>
```

Comment:
No language in path if I use wh_media_util:prompt_path(?WH_MEDIA_DB, PromptId)

(whistle_apps@kz527.onnet.su)4> wh_media_util:prompt_path(<<"system_media">>, <<"vm-saved">>).                                    
<<"/system_media/vm-saved">>